rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -104,10 +104,14 @@ def process(self, message, **kwargs):
                 entity = {"start": match["start"],
                           "end": match["end"],
                           "text": match["text"],
-                          "value": match["value"]["value"],
+                          "value": None,
                           "additional_info": match["value"],
                           "entity": match["dim"]}
-
+                try:
```

Comment:
Lets do the following:
- put that code block before L104 and the entity object creation and store the extracted value in a tmp var
- lets move that whole common code block to a function to be used by both components
- I'd rather not do the try and except but rather decide where to get the value from using the `match["value"]["type"] == "interval"`

Reply:
which file should I put the fucntion in ideally?